config/serializer/Node.yml

```diff
@@ -1,3 +1,6 @@
 Adshares\AdsOperator\Document\Node:
     exclusion_policy: ALL
     properties:
+        version:
+            expose: true
+            type: string
```

Comment:
This one is responsible for custom things (not fetching from ADS Client), so it can stay only here.

Reply:
Now I see